backend/.gitignore

```diff
@@ -0,0 +1,4 @@
+/node_modules
+/models
+/config
+/db.js
```

Comment:
the reason why the gitignore is not working is that u are using absolute paths. try something like this:
```
node_modules/
```

Reply:
I didn't can  resolve this after the changes.